conf/modules.config

```diff
@@ -25,9 +25,8 @@ process {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     */
 
-    withName: INDEX_TO_BED {
-        ext.args2   = '\'BEGIN {FS="\t"}; {print \$1 FS "0" FS \$2}\''
-        ext.suffix  = 'bed'
+    withName: MOSDEPTH {
+        ext.args = "--no-per-base --fast-mode --quantize 0:1:4:"
```

Comment:
```suggestion
        ext.args = "--no-per-base --fast-mode --quantize 0:1:4: --flag 1804 --mapq 1" 
```
Filter reads by flag and mapq

Reply:
I've also added the required env variables at the bottom of `modules.conf`

Reply:
To correctly determine variant calling regions, `quantized` bed should be filtered on regions with the `CALLABLE` tag and then intersected with an optional given ROI

Reply:
Allright thanks!